src/detect-http-uri.c

```diff
@@ -153,6 +153,13 @@ void DetectHttpUriRegister (void)
     DetectAppLayerMpmRegister2("http_raw_uri", SIG_FLAG_TOSERVER, 2, PrefilterGenericMpmRegister,
             GetRawData, ALPROTO_HTTP1, HTP_REQUEST_LINE);
 
+    // no difference between raw and decoded uri for HTTP2
```

Comment:
What do we want for this ?
I guess the good way to go is to test urlencoded uris against a real web server...